pkg/util/utils.go

```diff
@@ -112,19 +112,9 @@ func GetLocalkubeDownloadURL(versionOrURL string, filename string) (string, erro
 		// no 'v' prefix in input, need to prepend it to version
 		versionOrURL = "v" + versionOrURL
 	}
-	if k8sReleases, err := kubernetes_versions.GetK8sVersionsFromURL(constants.KubernetesVersionGCSURL); err != nil {
-		return "", errors.Wrap(err, "Error validating the localkube version")
-	} else {
-		isValidVersion := false
-		for _, version := range k8sReleases {
-			if version.Version == versionOrURL {
-				isValidVersion = true
-				break
-			}
-		}
-		if !isValidVersion {
-			return "", errors.New("Invalid localkube version")
-		}
+	isValidVersion, err := kubernetes_versions.IsValidLocalkubeVersion(versionOrURL, constants.KubernetesVersionGCSURL)
+	if !isValidVersion {
+		return "", errors.New("Not a valid localkube version to download")
```

Comment:
Oops, I missed a clause here, we should be checking the error too.  That should use `errors.Wrap`